### PR TITLE
Asset cleanup

### PR DESCRIPTION
**Pull request by ravenbuilder934**
Closes #
Closes #
Closes #

# Changes proposed in this pull request:
- Part of #6589 
- Removes unused assets
- Fixes some typos in texture names (`papper` instead of `paper`, etc)

------

### File size after: 32.2 MB

-------

Review please

### CODE REVIEW

File: src/main/resources/assets/minecolonies/gui/windowfilterablelist.xml [hunk 1]
```diff
@@ -1,26 +0,0 @@
-<window xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" size="190 244" pause="false"
```
Comment (Nightenom) [inline src/main/resources/assets/minecolonies/gui/windowfilterablelist.xml:1]: not sure about that

Comment (ravenbuilder934) [reply src/main/resources/assets/minecolonies/gui/windowfilterablelist.xml:1]: Doesn't seem like it gets called anywhere

**Review by Nightenom — COMMENTED** [review]
stop removing the pngs, or at least move them somewhere else (out of src folder)

File: src/main/resources/assets/minecolonies/gui/windowfilterablelist.xml (hunk 1 above)
Comment (Nightenom) [reply src/main/resources/assets/minecolonies/gui/windowfilterablelist.xml:1]: "doesn't" doesn't seem sure

**Comment by ravenbuilder934** [comment]
Where else can I move them?

File: src/main/resources/assets/minecolonies/gui/windowfilterablelist.xml (hunk 1 above)
Comment (ravenbuilder934) [reply src/main/resources/assets/minecolonies/gui/windowfilterablelist.xml:1]: I'm 95% sure it doesn't get called anywhere. Doing a search in files for `windowfilterablelist` has no results

**Comment by Nightenom** [comment]
> Where else can I move them?

I would create a folder something like unused resources, but up to others if they want it too

File: src/main/resources/assets/minecolonies/gui/windowfilterablelist.xml (hunk 1 above)
Comment (Asherslab) [reply src/main/resources/assets/minecolonies/gui/windowfilterablelist.xml:1]: it's not used anywhere
Screen Shot 2021-02-26 at 8 29 21 am

File: src/main/resources/assets/minecolonies/models/item/super_bread.json [hunk 2]
```diff
@@ -1,6 +0,0 @@
-{
-	"parent": "item/generated",
-	"textures": {
```
Comment (Raycoms) [inline src/main/resources/assets/minecolonies/models/item/super_bread.json]: @Mekle001 is this one unused?

**Comment by Raycoms** [comment]
We can remove the very old blockhut assets and duplicate assets that are in structurize, but the extra assets we have for our GUIs are useful and should be kept.

File: src/main/resources/assets/minecolonies/models/item/super_bread.json (hunk 2 above)
Comment (Mekle001) [reply src/main/resources/assets/minecolonies/models/item/super_bread.json]: Currently, yes. That can rapidly change, if somebody has an idea for effect.

**Comment by Mekle001** [comment]
Don't remove the forester recipes. They serve a purpose of removing that variant of the recipe on upgrade.

File: src/main/resources/assets/minecolonies/gui/windowfilterablelist.xml (hunk 1 above)
Comment (Nightenom) [reply src/main/resources/assets/minecolonies/gui/windowfilterablelist.xml:1]: Oh it's been moved to layouts

File: src/main/resources/assets/minecolonies/models/item/super_bread.json (hunk 2 above)
Comment (Nightenom) [reply src/main/resources/assets/minecolonies/models/item/super_bread.json]: that would probably fit moving to unused resources

**Review by Nightenom — COMMENTED** [review]
hey, the sounds :D again
minecolonies/textures/entity/skills/big/agility.png - don't remove those big icons
minecolonies/textures/gui/townhall_book_icon.png | turn_page_left/right.png
minecolonies/textures/icons/hunger_icon.png
otherwise good

**Comment by ravenbuilder934** [comment]
"minecolonies/textures/gui/townhall_book_icon.png | turn_page_left/right.png
minecolonies/textures/icons/hunger_icon.png"
What about these PNGs @Nightenom?

**Comment by Raycoms** [comment]
Please keep some of the texture UI assets, I think those shouldn't be too big and are useful having as an option.

**Comment by ravenbuilder934** [comment]
I did add some back https://github.com/ldtteam/minecolonies/pull/6611/commits/ce48bc2211031fb49ba7f7b99e9772c1a49abccc

**Comment by Nightenom** [comment]
> hey, the sounds :D again
> minecolonies/textures/entity/skills/big/agility.png - don't remove those big icons
> minecolonies/textures/gui/townhall_book_icon.png | turn_page_left/right.png
> minecolonies/textures/icons/hunger_icon.png
> otherwise good

@ravenbuilder934 hello there, how about these, if you need an explanation: big skills - rendering in world, the rest - general usability

**Comment by ravenbuilder934** [comment]
Yes but what *about* them? Do you want me to add them back where they were? Do you want me to put them in an `unusedAssets` folder?

**Comment by Nightenom** [comment]
your choice, if you want to reduce file size then unused folder is a solution

**Comment by ravenbuilder934** [comment]
done

**Review by Nightenom — APPROVED** [review]
last one textures/gui/townhall_book_title.png
hoping you didn't break any resource paths :smiley:

**Comment by ravenbuilder934** [comment]
Why are we saving that one? iirc it's just some text saying `Town Hall Book`

**Comment by Nightenom** [comment]
> Why are we saving that one? iirc it's just some text saying `Town Hall Book`

It's styled text, but welp up to you, just no need to pointlessly delete something

**Review by marchermans — APPROVED** [review]

**Comment by ravenbuilder934** [comment]
Ray...today is Saturday and you just merged a PR marked Merge on **Monday**

**Pull request merged**